src/item_factory.cpp

```diff
@@ -1821,12 +1822,25 @@ void Item_factory::load_toolmod( const JsonObject &jo, const std::string &src )
     }
 }
 
+// TODO: Refactor this with load_armor
+// This function does load_slog( def.tool ), but otherwise they are the same
```

Comment:
It may have been one, but I think this is a typo
```suggestion
// This function does load_slot( def.tool ), but otherwise they are the same
```

Reply:
Oops, thank you!